cmd/lambda/lease_auth/create_test.go

```diff
@@ -224,8 +223,8 @@ func TestGetLeaseAuth(t *testing.T) {
 
 		for _, tt := range tests {
 			t.Run(tt.name, func(t *testing.T) {
-				expectedLease := &db.Lease{}
-				expectedAccount := &db.Account{}
```

Comment:
linter prefers single-line assignments (and I do too, fwiw)

either way, I'd prefer to be consistent with this